VMBackup/main/hostsnapshotter.py

```diff
@@ -36,15 +36,27 @@
 from Utils import HostSnapshotObjects
 from Utils import HandlerUtil
 from fsfreezer import FsFreezer
+from dhcpHandler import DhcpHandler
 import sys
 
+
 class HostSnapshotter(object):
     """description of class"""
     def __init__(self, logger):
         self.logger = logger
         self.configfile='/etc/azure/vmbackup.conf'
-        self.snapshoturi = 'http://168.63.129.16/metadata/recsvc/snapshot/dosnapshot?api-version=2017-12-01'
-        self.presnapshoturi = 'http://168.63.129.16/metadata/recsvc/snapshot/presnapshot?api-version=2017-12-01'
+        self.dhcpHandlerObj = None
+        self.hostIp = '168.63.129.16'
+        try:
+            self.dhcpHandlerObj = DhcpHandler(self.logger)
+            self.hostIp = self.dhcpHandlerObj.getHostEndoint()
```

Comment:
Instead of finding hostIp in HostSnapshotter, it should happen before FsFreeze and come as input as one property of "paras" parameter in snapshotall() and pre_snapshot(). This will help in 
- Not adding any time delays in finding hostIP usinf dhcp during Freeze state
- Logging (to file) in dhcpHelper will not cause any issues during Freeze state,